migrations/20250319190617_add_childid_opoutputs.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.withSchema('dbos').table('operation_outputs', function (table) {
+    table.text('child_id');
```

Comment:
```suggestion
    table.text('child_workflow_id');
```
I think we need to be more specific about the ID.